ts/util/requests.ts

```diff
@@ -15,6 +15,16 @@ export function post(url: string, body: any) {
   });
 }
 
+export function get(base: string, body: string) {
+  let headers: any = {
+    "Content-Type": "application/json",
+  };
+  return fetch(`${base}?q=${encodeURIComponent(body)}`, {
```

Comment:
`get` should be able to handle any parameters - not just `q`. There should be a `parameters: Record<string, string>` parameter, so this:

```
get("https://dimsum.com/rest/test", {
  "q": "some text",
  "otherParam": 1234
});
```

results in: `GET https://dimsum.com/rest/test?q=some%20text&otherParam=1234`

You can use `new URLSearchParams(parmeters).toString()` to generate everything after the `?`